### PR TITLE
schema: fix items based on latest spec

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by dqminh** [comment]
LGTM

File: schema/config-linux.json [hunk 1]
```diff
@@ -245,6 +245,10 @@
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/kernel",
                                 "$ref": "defs.json#/definitions/int64Pointer"
                             },
+                            "kernelTCP": {
+                                "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/kernelTCP",
+                                "$ref": "defs.json#/definitions/int64Pointer"
```
Comment (wking) [inline schema/config-linux.json]: We definitely need a `kernelTCP` entry (catching up with #235), but after #555 I'm not sure we want to use pointers here.  The optional-ness is represented in JSON Schema by not listing these properties in a `required` array; we don't have to (or want to?) allow people to explicitly set a `null` value.

**Comment by mrunalp** [comment]
LGTM

**Comment by RobDolinMS** [comment]
@wking Can your comment be addressed in a separate PR?

**Comment by vbatts** [comment]
LGTM

**Pull request merged**